templates/ai/mlops.html

```diff
@@ -0,0 +1,210 @@
+        </div>
+        <p>Charmed Kubeflow is the foundation of Canonical MLOps. It integrates with various tools including Charmed MLFlow, Canonical Observability Stack and Charmed Spark covering all the stages of the machine learning lifecycle.</p>
+      </div>
+      <div class="p-section--shallow">
+        <hr>
+        <a class="p-button--positive" href="https://charmed-kubeflow.io/docs/get-started-with-charmed-kubeflow">Get started with Charmed Kubeflow</a>
+      </div>
+      <div class="u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/a59f849e-what is MLOps illustration.svg" alt="" style="height: 512px; width: 1309px;">
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h2 class="p-text--small-caps">MLOps services</h2>
+      </div>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Full support for your ML stack</h3>
+        </div>
+        <p>Canonical offers a fully supported machine learning operations solution, with guaranteed SLAs. Get the same level of support across public or private clouds.</p>
+        <p>
+          <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Charmed Kubeflow%20-%20Digital.pdf">Get the Charmed Kubeflow datasheet&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+      <div class="p-section--shallow">
+        <hr class="p-rule">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Managed MLOps</h3>
+        </div>
+        <p>We run the platform. Your team can focus on developing and deploying models. Streamline operational service delivery and offload the design, implementation and management of your MLOps environment.</p>
+        <p>
+          <a href="https://assets.ubuntu.com/v1/4e502b09-Managed.Kubeflow.DS.19.01.22+%281%29.pdf">Get the Managed MLOps datasheet&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+      <div class="p-section--shallow">
+        <hr class="p-rule">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Open source MLOps in the public cloud</h3>
+        </div>
+        <p>Get started with machine learning in the public cloud. Use the Charmed Kubeflow appliance on AWS to experiment with MLOps quickly and scale up with Canonical's support.</p>
+        <p>
+          <a href="https://aws.amazon.com/marketplace/pp/prodview-ssgryrrrydtds?sr=0-1&ref_=beagle&applicationId=AWSMPContessa">Get started with Kubeflow Appliance&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+      <div class="p-section--shallow">
+        <hr class="p-rule">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--2">Get your team up-to-date <br class="u-hide--small"> an MLOps workshop</h3>
```

Comment:
```suggestion
          <h3 class="p-heading--2">Get your team up-to-date <br class="u-hide--small">with a MLOps workshop</h3>
```
Missing a word and I think it should be 'a MLOps', will leave a comment in the doc

Reply:
@Sophie-32 can confirm, but I think it actually is supposed to be "an MLOps". Reasoning [here](https://www.scribbr.com/commonly-confused-words/a-vs-an/#:~:text=A%20or%20an%20before%20acronyms,-The%20same%20rule&text=Acronyms%20beginning%20with%20a%20consonant,vowel%20sound%20use%20%E2%80%9Can.%E2%80%9D)